rancher/dashboard
# Populates the Configurations input when deploying from Manifest

### Summary
Fixes https://github.com/epinio/ui/issues/133


### Occurred changes and/or fixed issues

- Added a new `URL searchParams` to know that we're deploying from a manifest.
- if the param is found, we set the deployment configurations list from the application configuration



### To test


- Download the manifest of one of the apps that has some configurations bonded to the app.
- Create a new deployment, click on From Manifest
- Follow the steps to the `Bindings` step
- The input should have the application service selected.


### Notes
- I noticed that the `manifest` has no reference to the namespace, when deploying from the manifest, it will select the first available namespace. If it happens that the given namespace does not have a given service, it won't be able to bind.
- With configurations alone, everything seems to work. If there are bonded services, while exported they will show as part of the configurations array (second element)
```js
"configurations":["dev-config","x4c6fceca75e456f25e7e99531e24-mysql"]
```
This seems to be a bug, we might need to tackle this issue 


### Screenshot/Video


![image](https://user-images.githubusercontent.com/88777903/212122434-4ccaf81f-543a-46ad-8a71-28e7faf2ca80.png)
